browser/brave_shields/ad_block_custom_resources_browsertest.cc

```diff
@@ -204,19 +204,19 @@ IN_PROC_BROWSER_TEST_F(AdblockCustomResourcesTest, AddEditRemoveScriptlet) {
 
   EXPECT_EQ("user-custom-script.js", GetCustomScriptletName(web_contents()));
   EXPECT_EQ(kContent, GetCustomScriptletContent(web_contents()));
-  SaveCustomScriptlet("custom-script-edited", kEditedContent);
+  SaveCustomScriptlet("Custom-Script-Edited", kEditedContent);
```

Comment:
Don't we need a new test here to check for lowercase vs uppercase (like in the screenshot you have in the PR)?

Reply:
Not really, but it's easy to add such one.